dependencies.yaml

```
@@ -303,7 +303,7 @@ dependencies:
           - psutil>=5.8.0
           - pytest>=6.2.4,<8.0.0a0
           - pytest-cov>=2.12.1
-          - pytest-lazy-fixture>=0.6.3
+          - pytest-lazy-fixtures
```

Do we want a lower bound here as well?

---

Thanks @jakirkham !
I can update it if needed.
I wonder how we can determine the lowest version to use here.

---

Perhaps `1.0.0` is good enough to start?

```suggestion
          - pytest-lazy-fixtures>=1.0.0
```

---

Great! Thank you! Will update this.